packages/theme/package.json

```diff
@@ -32,9 +32,13 @@
     "test": "backstage-cli package test"
   },
   "dependencies": {
-    "@material-ui/core": "^4.12.2"
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/material": "^5.12.2"
   },
   "peerDependencies": {
+    "@material-ui/core": "^4.12.2",
+    "@types/react": "^16.13.1 || ^17.0.0",
```

Comment:
@Rugvip  was this intentional? It reverts the fix introduced here: https://github.com/backstage/backstage/pull/17116